dap_prinz_green_jobs/pipeline/green_measures/occupations/soc_map.py

```diff
@@ -326,6 +343,9 @@ def find_most_likely_soc(
         2. Get the 4-digit SOCs of the good (>top_n_sim_threshold) matches in the top n most similar.
         3. If there are a few of these (>=minimum_n) and over a certain proportion (>minimum_prop) of these are the same at the 4 digit level - use this as the SOC.
                 This will return (soc, the job titles given for this same soc)
+
+        Returns data in the format ((soc_2020_6, soc_2020_4, soc_2010), job_title) or None
+        If pathway 1. (above) isn't true then the output will be ((None, soc_2020_4, None), job_title) and job_title will be a set of multiple
```

Comment:
as you're returning multiple soc name options is there one we should pick downstream for analysis? I just assume the first one?

Reply:
yes I think so